moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1773,6 +1773,10 @@ as the new values that correspond to the group */
   bool has_acceleration_;
   bool has_effort_;
 
+  /** \brief We at least 5 steps per trajectory for testing jump thresholds with computeCartesianPath. Otherwise
+   * testJointSpaceJump doens't work well. */
+  static const unsigned int min_steps_for_jump_thresh = 5;
```

Comment:
std::size_t
also a static const should be ALL_CAPS
move this to the .cpp file inside the namespace but outside the class